config/locales/ko.yml

```
@@ -226,10 +226,19 @@ ko:
       sign_up: "회원 가입"
       forgot_password: "비밀번호를 잊으셨나요?"
       resend_confirmation: "인증 메일을 다시 받으시겠어요?"
+      continue_with_google: "Google로 계속하기"
+      continue_with_apple: "Apple로 계속하기"
 
   profiles:
     show:
       settings: "설정"
+
+  users:
```

YAMLlint (1.38.0)

[error] duplication of key "users" in mapping

(key-duplicates)